eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -97,9 +98,16 @@ ms.service: $service
 }
 
 function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation) { 
-  $packageInfoJson = Get-Content $packageInfoJsonLocation -Raw
+  try {
```

Comment:
Since the specific case we're trying to handle here is the file not existing we could use: 

```pwsh
if (!TestFile $packageInfoJsonLocation) { 
  LogWarning "Package metadata not found for $packageInfoJsonLocation" 
  return 
} 
```

Using `return` instead of `exit` allows the loop that is calling `UpdateDocsMsMetadataForPackage` to continue executing.